src/components/AutocompleteSelectMenu/AutocompleteSelectMenu.tsx

```diff
@@ -80,7 +80,7 @@ const AutocompleteSelectMenu: React.FC<AutocompleteSelectMenuProps> = props => {
   } = props;
   const classes = useStyles(props);
 
-  const [inputValue, setInputValue] = React.useState(displayValue || "");
+  const [inputValue, setInputValue] = React.useState(displayValue || null);
```

Comment:
Is this change linked with the task in any way? If not, using `""` as default value for input does not produce any warnings in console

Reply:
Leaving  `""` there is the reason of the issue with placeholder text related to onBlur event. This is the second bug related to this PR.
I believe this part of the code is to handle no value case, not to deal with warnings or errors.

Reply:
This fixes one bug but creates another, try to pick an item without typing anything into the Input and you'll see that the issue still occurs.